packages/syft/setup.cfg

```diff
@@ -34,7 +34,7 @@ syft =
     forbiddenfruit==0.1.4
     gevent==21.12.0
     importlib_metadata==4.11.3
-    jax==0.3.5
+    jax==0.3.7
```

Comment:
We need to be careful with some of the newer dependencies as they are not available widely across different OS / cpu combinations.